build/yamls/antrea-aks.yml

```diff
@@ -3848,6 +3940,10 @@ data:
     # Enable controlling SNAT IPs of Pod egress traffic.
     #  Egress: false
 
+    # Enable flexible IPAM mode for Antrea. This mode allows to assign IP Ranges to Namespaces,
+    # Deployments and StatefulSets via IP Pool annotation.
+    #  FlexibleIPAM: false
```

Comment:
pedant comment: we have been using "flexible IPAM" in the scope of https://github.com/antrea-io/antrea/issues/1400 which is very different from what we are achieving here. I don't think we need to change anything there. We will probably recall PR 1400 "Optimized IPAM"

Reply:
Good point, or maybe we'll be able to combine both under `Flexible`.

Reply:
Maybe just call it AntreaIPAM?

Reply:
Sounds good to me